src/main/java/io/odpf/depot/config/BigQuerySinkConfig.java

```diff
@@ -39,6 +39,13 @@ public interface BigQuerySinkConfig extends OdpfSinkConfig {
     @Key("SINK_BIGQUERY_TABLE_PARTITION_KEY")
     String getTablePartitionKey();
 
+    @Key("SINK_BIGQUERY_TABLE_CLUSTERING_ENABLE")
+    @DefaultValue("false")
+    Boolean isTableClusteringEnabled();
+
+    @Key("SINK_BIGQUERY_TABLE_CLUSTERING_KEY")
+    String getTableClusteringKey();
```

Comment:
we can directly return List<String> here with separator.

   @Key("SINK_BIGQUERY_TABLE_CLUSTERING_KEY")
   @Separator(",")
   List<String> getTableClusteringKey();